src/sage/combinat/permutation.py

```diff
@@ -3094,6 +3094,76 @@ def is_derangement(self) -> bool:
         """
         return not self.fixed_points()
 
+    def is_simple(self) -> bool:
+        """
+        Return if ``self`` is simple.
+
+        A permutation is simple if it does not send any proper sub-interval
+        to a sub-interval.
+
+        See :oeis:`A111111`
+
+        For instance, ``[6,1,3,5,2,4]`` is not simple because it maps the
+        interval ``[3,4,5,6]`` to ``[2,3,4,5]``, whereas ``[2,6,3,5,1,4]`` is
+        simple.
+
+        EXAMPLES::
+
+            sage: g = Permutation([4,2,3,1])
+            sage: g.is_simple()
+            False
+
+            sage: g = Permutation([6,1,3,5,2,4])
+            sage: g.is_simple()
+            False
+
+            sage: g = Permutation([2,6,3,5,1,4])
+            sage: g.is_simple()
+            True
+
+        TESTS::
+
+            sage: [len([pi for pi in Permutations(n) if pi.is_simple()])
```

Comment:
Why is this only a TEST? It is relevant for the user (not the most efficient way to get the list of simple permutations though).

Reply:
Précisément, c'est un lien, il faut cliquer dessus pour avoir l'info.

Reply:
Voilà. On cythonise ou pas ? J'ai la flemme, et d'autres occupations plus pressantes.

Reply:
Je vois pas trop l'intérêt à ce stade.